pytorch_blade/torch_blade/clustering/support_fusion_group.py

```diff
@@ -16,7 +16,7 @@
 from torch_blade.clustering.support_fusion_algorithm import group_supported_clusters
 from torch_blade.logging import logger
 
-_MIN_GROUP_NODES = 3
+_MIN_GROUP_NODES = 1
```

Comment:
is it safe to change the min cluster here?  If we only want to test the UT, better not change it by default without benchmarking.